File: src/main/java/client/database/data/AccountData.java
package client.database.data;

public class AccountData {
   private int id;

   private String name;

   private String password;

   private byte gender;

   private boolean banned;

   private String pin;

   private String pic;

   private byte characterSlots;

   private byte tos;

   private int language;

   public AccountData(int id, String name, String password, byte gender, boolean banned, String pin, String pic, byte characterSlots, byte tos, int language) {
      this.id = id;
      this.name = name;
      this.password = password;
      this.gender = gender;
      this.banned = banned;
      this.pin = pin;
      this.pic = pic;
      this.characterSlots = characterSlots;
      this.tos = tos;
      this.language = language;
   }

   public int getId() {
      return id;
   }

   public String getName() {
      return name;
   }

   public String getPassword() {
      return password;
   }

   public byte getGender() {
      return gender;
   }

   public boolean isBanned() {
      return banned;
   }

   public String getPin() {
      return pin;
   }

   public String getPic() {
      return pic;
   }

   public byte getCharacterSlots() {
      return characterSlots;
   }

   public byte getTos() {
      return tos;
   }

   public int getLanguage() {
      return language;
   }
}

File: engine/model/src/main/scala/client/database/data/AccountData.scala
package client.database.data

class AccountData(private var _id: Int, private var _name: String, private var _password: String,
                  private var _gender: Byte, private var _banned: Boolean, private var _pin: String,
                  private var _pic: String, private var _characterSlots: Byte, private var _tos: Byte,
                  private var _language: Int) {
  def id: Int = _id

  def name: String = _name

  def password: String = _password

  def gender: Byte = _gender

  def banned: Boolean = _banned

  def pin: String = _pin

  def pic: String = _pic

  def characterSlots: Byte = _characterSlots

  def tos: Byte = _tos

  def language: Int = _language
}
